src/Eccube/Controller/Admin/Store/OwnerStoreController.php

```diff
@@ -159,6 +159,7 @@ public function search(Application $app, Request $request)
             'items' => $items,
             'promotionItems' => $promotionItems,
             'message' => $message,
+            'url_detail' => $this->appConfig['owners_store_detail'],
```

Comment:
not need this parameter.
@see https://github.com/EC-CUBE/ec-cube/pull/2614/files#diff-52e8599ef66e95532b0449717d83314cR46